manifests/agent.pp

```diff
@@ -289,25 +289,34 @@
     }
   }
 
-  # Ensure that the correct config file is used.
+  # Ensure that the correct startup file is used.
   if $manage_startup_script {
+    $service_name = $servicename ? {
```

Comment:
selector statements <3

Reply:
Isn't that kind of breaking change? 
Users were able to override service name before, now we default to either `zabbix-agent` or `zabbix-agent2`

Reply:
Users select the servicename by setting $servicename and there's no change there.
The variable $service_name is only used when calling startup.pp. The parameter sent to startup.pp was earlier hardcoded to 'zabbix-agent'.

Reply:
oops you're right